app/models/hackathon/scheduled.rb

```diff
@@ -26,17 +26,17 @@ def end_date
 
   def dates_are_chronological
     if ends_at < starts_at
-      errors.add(:ends_at, :before_the_start)
+      errors.add(:ends_at, "must be after the start time")
     end
   end
 
   def dates_are_in_the_future
     if starts_at < Time.now
-      errors.add(:starts_at, :in_the_past)
+      errors.add(:starts_at, "must be in the future")
     end
 
     if ends_at < Time.now
-      errors.add(:ends_at, :in_the_past)
+      errors.add(:ends_at, "must be in the future")
```

Comment:
The second argument to `add` is `type`, which should be a symbol. See the [docs](https://api.rubyonrails.org/classes/ActiveModel/Errors.html#method-i-add)

Example:
```ruby
errors.add(:ends_at, :non_chronological, message: "must be after the start time")
```

The same thing applies to:
```ruby
errors.add(:base, :already_subscribed, message: "You've already subscribed to this area!")
```